src/www/components/popup/PopupComp.jsx

```diff
@@ -0,0 +1,183 @@
+import "./PopupComp.scss";
+import React, { Component } from "react";
+
+import Input from "../../components/input/Input";
+import axios from "axios";
+import Popup from "reactjs-popup";
+import ImageUploader from "react-images-upload";
+import ScrollArea from "react-scrollbar";
+import { Redirect } from "react-router-dom";
+import { withTranslation } from "react-i18next";
+
+class PopupComp extends Component {
+  state = {
+    teamName: "",
+    description: "",
+    teamCategories: [],
+    teamPicture: "",
+    serverResponse: "",
+    redirect: false,
+  };
+
+  createTeam = (name, description, categories, photo) => {
+    axios
+      .post("/api/teams/create", {
+        name,
+        description,
+        categories,
+        photo,
```

Comment:
We cannot upload the picture in one call (apparently?), so we should chain those requests one after another.